files/default/neutron-ha-tool.py

```diff
@@ -240,15 +240,14 @@ def run(args):
     qclient.format = 'json'
 
     if args.agent_selection_mode == 'random':
-        Configuration.agent_picker_class = RandomAgentPicker
+        Configuration.agent_picker = RandomAgentPicker()
```

Comment:
Yes, this is much better.  As [previously mentioned](https://github.com/SUSE-Cloud/cookbook-openstack-network/pull/9#discussion_r94273578) there's still a code smell since you're using a class variable to store the instance of the strategy subclass.  I would prefer it if you instantiated from the `Configuration` class and then passed that configuration object around to whatever needs it.  That might even result in showing that a separate configuration class is not needed, since the only configuration currently is which strategy subclass instance needs to be used.

BTW I can live with it, but I still wanted to be annoying and point out that this commit goes directly against [part of my advice for how to amend PRs](http://blog.adamspiers.org/2015/03/24/why-and-how-to-correctly-amend-github-pull-requests/#sec-2).